challenges/prototypes.js

```diff
@@ -5,29 +5,47 @@
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
+function CuboidMaker(options){
+    this.length = options.length;
+    this.width = options.width;
+    this.height = options.height;
+  }
+
 
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
   
   Formula for cuboid volume: length * width * height
 */
-
+CuboidMaker.prototype.volume = function volume() {
+  return `${this.length * this.width * this.height}`;
+}; 
 
 /* == Step 3: Surface Area Method ==
   Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
-
+CuboidMaker.prototype.surfaceArea = function surfaceArea() {
```

Comment:
make sure to read everything, you left off part of the equation and that is why the result was not the same 
Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
so you should have this
return `Surface area: ${2 * (this.length * this.width + this.length * this.height + this.width * this.height)}`;

Reply:
got it, fixed it!